From 575c96e1864cd32197e8aeff0e8370bed31f256e Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 22 Jul 2014 14:36:56 -0400
Subject: [virt] kvm_host: typo fix

Message-id: <1406040016-3289-22-git-send-email-pbonzini@redhat.com>
Patchwork-id: 86100
O-Subject: [RHEL7 PATCH v2 021/221] kvm_host: typo fix
Bugzilla: 1116936
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: "Michael S. Tsirkin" <mst@redhat.com>

fix up typo in comment.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit 81e87e26796782e014fd1f2bb9cd8fb6ce4021a8)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/kvm_host.h b/include/linux/kvm_host.h
index fa03d6e..b41b4e5 100644
--- a/include/linux/kvm_host.h
+++ b/include/linux/kvm_host.h
@@ -803,7 +803,7 @@ static inline void kvm_guest_enter(void)
 
  /* KVM does not hold any references to rcu protected data when it
   * switches CPU into a guest mode. In fact switching to a guest mode
-  * is very similar to exiting to userspase from rcu point of view. In
+  * is very similar to exiting to userspace from rcu point of view. In
   * addition CPU may stay in a guest mode for quite a long time (up to
   * one time slice). Lets treat guest mode as quiescent state, just like
   * we do with user-mode execution.
-- 
1.7.1